guides/source/engines.md

```diff
@@ -1505,7 +1505,7 @@ becomes
 ActiveSupport.on_load(:active_record) { self.include_root_in_json = true } # self refers to ActiveRecord::Base here
 ```
 
-## Available Hooks
+### Available Hooks
```

Comment:
This is probably a different section and not under above one

Reply:
I thought they describe all the hooks that can be called with `on_load`. The available hooks list all the frameworks.

Reply:
Maybe that text should be changed as well. For example `action_cable` is not a hook, but an argument that can be passed to `on_load`